### PR TITLE
Fix/update css classes to fit convention

### PR DESCRIPTION
**Pull request by mrvary**
## PR to streamline working with css and components

- [x] move every component into its own file
  - [x] ButtonRow -> Competition
  - [x] Button component ( `mode={primary / secondary}` uses primary/secondary color )
  - [x] ... App.js should only consist of one component 
- [x] rename css classes to fit the components, e.g. `app__element`
- [x] refactor naming of elements
- [x] use global color variables

### CODE REVIEW

**Comment by WillKR** [comment]
schön verschönert

File: src/react/components/Footer.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.footer__container {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.footer-text {
```
Comment (WillKR) [inline src/react/components/Footer.css]: little bug

> .footer__text

like in Footer.js

**Comment by WillKR** [comment]
rewritten as discussed

File: electron/main.js [hunk 2]
```diff
@@ -1,24 +1,24 @@
 // node dependencies
-const path = require('path');
```
Comment (Ceecker51) [inline electron/main.js]: Do we use single or double quotes?

Comment (mrvary) [reply electron/main.js]: double quotes :)

**Pull request merged**